libr/core/linux_heap_jemalloc.c

```diff
@@ -372,6 +372,89 @@ static void GH(jemalloc_get_bins)(RCore *core, const char *input) {
 	}
 }
 
+static void GH(jemalloc_get_runs)(RCore *core, const char *input) {
+	switch (input[0]) {
+	case ' ':
+		{
+			int i, j;
+			GHT addr = UT64_MAX, size = UT64_MAX, mapbits = UT64_MAX, flags = UT64_MAX, mask = UT64_MAX, chunk = UT64_MAX, misc_offset=UT64_MAX;
+			GHT map = UT64_MAX, map_misc_offset = UT64_MAX, chunksize_mask = UT64_MAX, npages = UT64_MAX, map_bias = UT64_MAX;
+			arena_chunk_t *c = R_NEW0 (arena_chunk_t);
+
+			input += 1;
+			chunk = strstr (input, "0x") ? (GHT)strtol (input, NULL, 0) : (GHT)strtol (input, NULL, 16);
+
+			if (GH(r_resolve_jemalloc)(core, "je_chunk_npages", &npages)) {
+				r_core_read_at (core, npages, (ut8*)&npages, sizeof (GHT));
+			} else {
```

Comment:
handle error in the first branch, and put the core_read_at below outside the else block, return blocks doesnt need an else